cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -1004,15 +1260,10 @@ Schannel::SSLEngine::newCredentialsHandle(bool incoming) const
         // Don't set SCH_SEND_ROOT_CERT as it seems to cause problems with Java certificate validation and Schannel
         // doesn't seems to send the root certificate either way.
         cred.dwFlags = SCH_CRED_NO_SYSTEM_MAPPER | SCH_CRED_DISABLE_RECONNECTS;
-
-        // There's no way to prevent Schannel from sending "CA names" to the client. Recent Windows versions don't CA
-        // names but older ones do send all the trusted root CA names. We provide the root store to ensure that for
-        // these older Windows versions, we also include the CA names of our trusted roots.
-        cred.hRootStore = _rootStore;
```

Comment:
The transceiver does this with the configured trusted certificates. We don't want to use this `_rootStore` when using options API.